### ISSUE TITLE
A question about bn_3d

### ISSUE THREAD

**Issue by jamiechoi1995** [Sense-X/UniFormer#67]
https://github.com/Sense-X/UniFormer/blob/1ce70bcccbe72962813aedf5eb1209f318f859b6/video_classification/slowfast/models/uniformer.py#L47-L48

I wonder why bn_3d if defined out of the model class?
In my opinion, a more general way is to define it inside the model, like
```python
self.bn_3d = nn.BatchNorm3d(dim)
```

**Comment by Andy1621** [comment]
Thanks for your suggestion. I use these definitions for clear code~~

**Comment by Andy1621** [comment]
As there is no more activity, I am closing the issue, don't hesitate to reopen it if necessary.

**Comment by jamiechoi1995** [comment]
> Thanks for your suggestion. I use these definitions for clear code~~

Thank you for your reply, the reason I ask this question is when I tried to modify all the normalization to batchnorm3d and defined it inside the model, I got "TypeError: expected CPU (got CUDA)". I wonder if it needs to be defined outside to avoid this error. Since you use these definitions for clear code, I will continue to debug the real reason.
